binding_generator.py

```diff
@@ -661,7 +661,7 @@ def generate_builtin_class_header(builtin_api, size, used_classes, fully_used_cl
             result.append(method_signature)
 
     # Move constructor.
-    result.append(f"\t{class_name}({class_name} &&other);")
+    result.append(f"\t{class_name}({class_name} &&p_other);")
```

Comment:
Some of these hard coded ones were changed as the implementations were dynamic and applied the modification, didn't touch the rest of the cases because they are largely copied directly from the engine side and that might be better to handle separately